packages/remix-dev/vite/plugin.ts

```diff
@@ -130,9 +130,11 @@ const CLIENT_ROUTE_EXPORTS = [
   "shouldRevalidate",
 ];
 
-// The "=1" suffix ensures client route requests can be processed before hitting
-// the Vite plugin since "?client-route" can be serialized as "?client-route="
-const CLIENT_ROUTE_QUERY_STRING = "?client-route=1";
+/** This is used to manage a build optimization to remove unused route exports
+from the client build output. This is important in cases where custom route
+exports are only ever used on the server. Without this optimization we can't
+tree-shake any unused custom exports because routes are entry points. */
+const BUILD_CLIENT_ROUTE_QUERY_STRING = "?__remix-build-client-route";
```

Comment:
Since this query string is no longer visible in the browser's network tab, I'm not trying to make this look pretty anymore and instead I'm aiming for it to be more explicit.

Reply:
Also the `=1` suffix is no longer needed since that was to solve an issue when proxying during development. In the production builds this query string never goes over the network.